guide/english/python/files-and-io/index.md

```diff
@@ -106,6 +106,16 @@ Sample code to open a text file using the `with` statement:
 with open('hello_world.txt', 'w') as f:
     f.write('Hello World!')
 ```
+#### Reading files
+There are many ways of reading a file. For a file <code>fh</code> opened in the read mode, the following are the options:
```

Comment:
Instead of having `opened in read mode` make it `reading a file is only possible if file is open in read mode` as it causes a sense of confusion and the text should be clear that reading is only possible in read mode.